GSA_Adapter/GSA_Adapter.csproj

```diff
@@ -102,6 +102,8 @@
     <Compile Include="AdapterActions\Pull.cs" />
     <Compile Include="AdapterActions\Execute.cs" />
     <Compile Include="CRUD\ReadResults.cs" />
+    <Compile Include="CRUD\ReadResultsElements.cs" />
+    <Compile Include="CRUD\ReadResultsGlobal.cs" />
```

Comment:
I appreciate this new subdivision of the methods instead of having everything in ReadResults